component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/smsotp/SMSOTPUtils.java

```diff
@@ -105,9 +105,10 @@ public static boolean isSMSOTPDisableForLocalUser(String username, Authenticatio
      * @param username  the Username
      * @param attribute the Attribute
      * @throws SMSOTPException
+     * @throws UserStoreException
      */
     public static void updateUserAttribute(String username, Map<String, String> attribute, String tenantDomain)
-            throws SMSOTPException {
+            throws SMSOTPException, UserStoreException {
```

Comment:
we cannot change public method signatures

Reply:
Is this Util class inside sms otp module will be used by other modules ? If not I have changed all the places which is using this method to handle the exceptions throwing